### ISSUE TITLE
HP Borrowing Clarification

### ISSUE THREAD

**Issue by bitfort** [mlcommons/training_policies#209]
Clarify HP borrow for v0.6.

**Comment by bitfort** [comment]
SWG Notes:

You can borrow hyper parameters during review. 

At a high level, we want to measure system performance not ability to tune hyperparameters. The goal of borrowing is enable submitters to focus on system performance instead of HP tuning. 

For hyper parameter borrowing, the basic idea is to "even you up (in terms of epochs to converge)" with the benchmark you are borrowing from, not improving upon them (in terms of epochs to converge). Specifically, at the same "scale" we expect everyone to converge in the same number epochs -- and "borrowing" is intended to enable this (among other objectives). 

We understand that sometimes borrowing can require re-turning in some situations; we do not intend borrowing to enable arbitrary re-tuning to improve performance. We do not have an exact way of drawing this line right now, however. 

When ever possible, we prefer HPs be borrowed as batch -- however this is complicated, especially for HPs which are orthogonal to optimizer/batch/learning rate. 

There is some burden on the borrower provide justification and data. 

There are more detailed ideas discussed in the special topics notes.
